cmd/internal/cli/instance_linux.go

```diff
@@ -38,6 +38,8 @@ func init() {
 	cmdManager.RegisterSubCmd(InstanceCmd, InstanceStartCmd)
 	cmdManager.RegisterSubCmd(InstanceCmd, InstanceStopCmd)
 	cmdManager.RegisterSubCmd(InstanceCmd, InstanceListCmd)
+
+	cmdManager.RegisterFlagForCmd(&commonEncryptFlag, InstanceStartCmd)
```

Comment:
Just to be consistent with other `instance start` flags, you could just replace `actionsCmd` https://github.com/sylabs/singularity/blob/master/cmd/internal/cli/action_flags.go#L675 by `actionsInstanceCmd`